meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio-mfgtool/imx8mmevk/lmp.cfg

```diff
@@ -9,7 +16,7 @@ CONFIG_SPL_LOAD_FIT=y
 CONFIG_OF_LIBFDT_OVERLAY=y
 CONFIG_FIT=y
 CONFIG_FIT_SIGNATURE=y
-CONFIG_FIT_SIGNATURE_STRICT=y
```

Comment:
We do want this enabled actually.  It's a local change we have to support stopping the boot if fit image does not pass signature checks.

Reply:
Yeah, strict is the only option we have to make sure boot can only succeed if signature is present and valid.

Reply:
sure ,will fix

Reply:
Should I add also `CONFIG_SPL_FIT_SIGNATURE_STRICT=y` for SPL ?

Reply:
Yup, strict should be enabled on both.